SPECS/libjpeg-turbo/libjpeg-turbo.spec

```diff
@@ -42,11 +45,8 @@ make %{?_smp_mflags}
 cd build
 make DESTDIR=%{buildroot} install
 
-%post
-/sbin/ldconfig
-
-%postun
-/sbin/ldconfig
+%post -p /sbin/ldconfig
```

Comment:
Out of curiosity: do you know if we're planning to remove these in a similar way [Fedora did it for versions 28+](https://fedoraproject.org/wiki/Changes/Removing_ldconfig_scriptlets)?

Reply:
There are no immediate plans, but dev does support the `%ldconfig_*` macros now.